build.gradle

```diff
@@ -1,54 +0,0 @@
-plugins {
-	alias(libs.plugins.versions)
-	alias(libs.plugins.nexusPublish)
-}
-
-group = "com.teamscale"
-
-ext {
-	appVersion = '29.1.3'
-
-	jacocoVersion = libs.versions.jacoco.get()
-
-	if (VersionUtils.INSTANCE.isTaggedRelease()) {
-		snapshotVersion = appVersion
-	} else {
-		snapshotVersion = appVersion + '-SNAPSHOT'
-	}
-}
-
-allprojects {
-	version snapshotVersion
-}
-
-// Installs all Maven artifacts to your local Maven repository
-task publishToMavenLocal
-subprojects {
-	// must be run after evaluation because the publishToMavenLocal tasks are generated by our plugin during
-	// project evaluation
-	afterEvaluate {
-		def publishTask = it.tasks.findByPath("publishToMavenLocal")
-		if (publishTask != null) {
-			rootProject.tasks.publishToMavenLocal.dependsOn(publishTask)
-		}
-	}
-}
-
-nexusPublishing {
-	repositories {
-		sonatype()
-	}
-}
-
-dependencyUpdates.resolutionStrategy {
```

Comment:
This was configuration for the `./gradlew dependencyUpdates` (from https://github.com/ben-manes/gradle-versions-plugin) task that lists all available updates for dependencies, but none of the settings had any effect at the moment, because not rc, beta, etc. updates were suggested.